versions/1.0.0.md

```diff
@@ -480,6 +487,91 @@ criteria:
     - $statusCode == 503
 ```
 
+#### Component Object
+
+Holds a set of reusable objects for different aspects of the Workflows Specification. All objects defined within the components object will have no effect on the workflow unless they are explicitly referenced from properties outside the components object.
+
+##### Fixed Fields
+
+Field Name | Type | Description
+---|:---|---
+<a name="componentInputs"></a> inputs | Map[`string`, [Schema Object](#schemaObject)] | An object to hold reusable JSON Schema objects to be referenced from workflow inputs.
+<a name="componentParameters">parameters</a> | Map[string, [Parameter Object](#parameter-object)] | An object to hold reusable Parameter Objects
+
+This object MAY NOT be extended with [Specification Extensions](#specificationExtensions).
+
+All the fixed fields declared above are objects that MUST use keys that match the regular expression: `^[a-zA-Z0-9\.\-_]+$`. The key is used to refer to the input or parameter in other parts of the Workflow document.
+
+Field Name Examples:
+
+```
+User
+User_1
+User_Name
+user-name
+my.org.User
+```
+
+##### Component Object Example
+
+```json
+"components": {
+  "inputs": {
+    "storeId": {

```

Comment:
These examples seem to be backwards no?  Isn't input a JSON Schema and parameter has the the name, in value property?

Reply:
Hmm - components.inputs is a Map of string to JSON Schema object, so I'm reasonably sure that piece is right (in this case `storeId` is the key). Not sure I understand the problem with parameter.